mod_cluster/WFLY-6803_Add_multi-server_support_to_mod_cluster.adoc

```diff
@@ -0,0 +1,66 @@
+= Title
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          clustering,undertow,mod_cluster
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Undertow subsystem supports defining and running multiple server instances (each consisting of listeners, hosts, servlet container, etc).
+At the time of writing, mod_cluster subsystem registers only the default server returned by `org.wildfly.extension.undertow.UndertowService#getDefaultServer`
+to register with the proxy servers.
+Different servers might be registered with different groups of proxy servers.
```

Comment:
This means that multiple independent proxies may defined on one EAP instance running on different Undertow `server`s?